lib/ads/client_side_ad_manager.js

```diff
@@ -48,6 +49,10 @@ shaka.ads.ClientSideAdManager = class {
     // the page.
     this.adsLoader_ = new google.ima.AdsLoader(adDisplayContainer);
 
+    if (locale != '') {
```

Comment:
Once the locale is set to navigator.language by default, you won't need this check anymore.

Reply:
I changed it!